src/views/Timeline.vue

```diff
@@ -427,6 +456,11 @@ export default {
       const p = Number(number).toLocaleString(undefined, { style: 'percent', minimumFractionDigits: 0 })
       return p
     }
+  },
+  watch: {
+    peerConnectionStatus: function (status) {
```

Comment:
@vickywane how about we pull all this logic in NavBar or a separate top level AppNotification component?

Reply:
That would be okay, however, I thought we could make the Snackbar more reusable by handling this logic in the parent components then passing in only props.